src/Core/Models/Business/OrganizationLicense.cs

```diff
@@ -135,6 +136,7 @@ public OrganizationLicense()
     public bool UseSecretsManager { get; set; }
     public int? SmSeats { get; set; }
     public int? SmServiceAccounts { get; set; }
+    public bool LimitCollectionCreationDeletion { get; set; }
```

Comment:
This might need a default value of `true`. i.e.

```suggestion
    public bool LimitCollectionCreationDeletion { get; set; } = true;
```

Think of the situation where a self-hosted server updates its code and has the flag on (so we'll use this value), but it's using an existing (old) license file that doesn't have this value in it. In that case, this class will be instantiated and this will default to `false`, which will then overwrite the default value of `true` in the db.

This is part of the trouble of having `true` as a default value instead of `false`. Won't be doing that again for sure.